src/wp-includes/class-wp-dependencies.php

```diff
@@ -199,10 +199,31 @@ public function all_deps( $handles, $recursion = false, $group = false ) {
 				continue;
 			}
 
-			$keep_going = true;
+			$keep_going           = true;
+			$missing_dependencies = array();
+			if ( isset( $this->registered[ $handle ] ) && $this->registered[ $handle ]->deps ) {
```

Comment:
Using an array in a boolean context works, but this is more explicit:

```suggestion
			if ( isset( $this->registered[ $handle ] ) && count( $this->registered[ $handle ]->deps ) > 0 ) {
```